api/proto/config/appconfig.proto

```diff
@@ -58,4 +58,7 @@ message AppInstanceConfig {
 
   // contains the encrypted userdata
   CipherBlock cipherData = 13;
+
+  // Volumes to be attached to the app-instance.
+  repeated string volumeRefs = 14;
```

Comment:
I would make sense to add a comment before the Drive above that it will be replaced by volumeRefs. That EVE receiving the message will, if it supports volumeRefs, ignore the Drive.

Reply:
added the comment.